app/views/users/new.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Create An Account</h1>
+
+<%= render partial: "form" %>
```

Comment:
- [ ] Please, I need you to leave the previous `form` and `form_tag` you created, just so we can check if you followed the tutorial correctly. They should just stay here **commented**.